app/routers/tdee.py

```diff
@@ -0,0 +1,40 @@
+from app.database import get_db
+from sqlalchemy.orm import Session
+from fastapi import APIRouter, Depends, HTTPException
+from app import models
+
+# Create a router for operations with 'users' request
+router = APIRouter(prefix="/tdees", tags=["tdees"])
+
+
+# Request to calculate  total daily energy expenditure
+@router.post("/{user_id}")
+def calculate_tdee(user_id: int, db: Session = Depends(get_db)):
+
+    user_query = db.query(models.User).filter(models.User.user_id == user_id)
+    user = user_query.first()
+
+    if not user:
+        raise HTTPException(status_code=404, detail="User not found")
+
+    if user.gender == "male":
+        basal_metabolic_rate = 10 * user.weight + 6.25 * user.height - 5 * user.age + 5
+    else:
+        basal_metabolic_rate = 10 * user.weight + 6.25 * user.height - 5 * user.age - 161
+
+    if user.activity_level == 1:
```

Comment:
instead of using multiple if-else statements, you can create a table or a dictionary {activity_level: tdee_coefficient} and then calculate tdee_number as following:
tdee_number = basal_metabolic_rate * activity_level_to_tdee_coeff[activity_level]